Hymns/SceneDelegate.swift

```diff
@@ -19,7 +19,13 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
         // Create the SwiftUI view and set the context as the value for the managedObjectContext environment keyPath.
         // Add `@Environment(\.managedObjectContext)` in the views that will need the context.
+        #if !DEBUG
+        let rootView = LaunchRouterView().environment(\.managedObjectContext, context)
+        #endif
+
+        #if DEBUG
         let rootView = HomeContainerView().environment(\.managedObjectContext, context)
+        #endif
```

Comment:
1) You can do use #else for the else case instead of if DEBUG and if !DEBUG
2) This thing makes me nervous. Since we usually develop on the debug build, we won't ever catch any errors that appear for the LaunchRouterView and the animation stuff until it launches to production. Could you find another way to determine if you're in a test or not? You can try to inspect `CommandLine.arguments` or something to see if there's any way we can tell between a debug test build and a debug dev build.

Reply:
Actually, it looks like you can add your own preprocessor macro to the build settings: https://stackoverflow.com/questions/242817/xcode-preprocessor-macros

So you can eventually just do
#if TEST
...
#else
...
#endif

Reply:
Okay, according to this stackoverflow preprocessor won't work  for this. https://stackoverflow.com/questions/6748087/xcode-test-vs-debug-preprocessor-macros 

but I did come up with a solution, that I think will work. It checks if testing instead of Debug and then routes accordingly.